flink-filesystems/flink-gs-fs-hadoop/pom.xml

```diff
@@ -188,6 +212,29 @@ under the License.
 							<goal>shade</goal>
 						</goals>
 						<configuration>
+							<filters>
+								<filter>
+									<artifact>org.apache.flink:flink-fs-hadoop-shaded</artifact>
+									<excludes>
+										<exclude>com/google/common/**</exclude>
+										<exclude>org/checkerframework/**</exclude>
+										<exclude>com/google/errorprone/**</exclude>
+										<exclude>com/google/j2objc/**</exclude>
+										<exclude>com/google/thirdparty/publicsuffix/**</exclude>
+									</excludes>
+								</filter>
+								<filter>
+									<artifact>*</artifact>
+									<excludes>
+										<exclude>properties.dtd</exclude>
+										<exclude>PropertyList-1.0.dtd</exclude>
+										<exclude>mozilla/**</exclude>
+										<exclude>META-INF/maven/**</exclude>
+										<exclude>META-INF/NOTICE.txt</exclude>
+										<exclude>META-INF/LICENSE.txt</exclude>
+									</excludes>
+								</filter>
+							</filters>
```

Comment:
We shouldn't exclude these dependencies during shading, but exclude them from when we're adding the dependency.

Reply:
`flink-fs-hadoop-shaded` is a shaded jar which doesn't relocate the classes mention in the exclusion rules here. The only way I see we can exclude them is by mentioning them explicitly here.